backend/libraries/types/src/file.rs

```diff
@@ -8,6 +8,7 @@ pub struct FileAdded {
     pub owner: UserId,
     pub hash: Hash,
     pub size: u64,
+    pub timestamp: TimestampMillis,
```

Comment:
serde default?

Reply:
Ah yes, I thought these were never stored in the stable state, but they are in the events queue which get persisted during upgrades.

Reply:
Done